<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>snes-apu-dbg</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QWidget" name="ramViewerContainer" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="voiceViewer0Container" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>140</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="voiceViewer1Container" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>140</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="voiceViewer2Container" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>140</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="voiceViewer3Container" native="true">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>140</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="voiceViewer6Container" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>170</y>
      <width>140</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="voiceViewer7Container" native="true">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>170</y>
      <width>140</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="voiceViewer4Container" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>140</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="voiceViewer5Container" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>170</y>
      <width>140</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="fileOpenSpcFile"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="fileOpenSpcFile">
   <property name="text">
    <string>&amp;Open SPC File...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
